Source/WTF/wtf/PlatformEnableGlib.h

```diff
@@ -65,4 +65,5 @@
 
 #if ENABLE(WPE_PLATFORM) || PLATFORM(GTK)
 #define ENABLE_DAMAGE_TRACKING 1
+#define ENABLE_MANAGED_MEDIA_SOURCE 1
```

Comment:
Don't we need an ifdef in `defaultManagedMediaSourceEnabled()` ?

Reply:
Where is this `#define` read?

Reply:
ENABLE(FOO) expands to ENABLE_FOO